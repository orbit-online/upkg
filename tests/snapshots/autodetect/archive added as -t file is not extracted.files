├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .bin
│   │   └── [lrwxrwxrwx]  acme.tar -> ../.packages/acme.tar+x@2fc13e1819bc3ed22ee1526d4f44ec20cd8748575e52a28e137f74fec2dd1160
│   ├── [drwxr-xr-x]  .packages
│   │   └── [-rwxr-xr-x]  acme.tar+x@2fc13e1819bc3ed22ee1526d4f44ec20cd8748575e52a28e137f74fec2dd1160
│   └── [lrwxrwxrwx]  acme.tar -> .packages/acme.tar+x@2fc13e1819bc3ed22ee1526d4f44ec20cd8748575e52a28e137f74fec2dd1160
└── [-rw-r--r--]  upkg.json
