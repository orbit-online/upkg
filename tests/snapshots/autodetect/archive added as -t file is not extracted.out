upkg: No checksum given for '$BATS_RUN_TMPDIR/package-fixtures/default/acme.tar', determining now
upkg: Added '$BATS_RUN_TMPDIR/package-fixtures/default/acme.tar'
